examples/arc/app.js

```diff
@@ -26,13 +25,13 @@ class Root extends Component {
       selectedCounty: null
     };
 
-    requestJson(DATA_URL, (error, response) => {
-      if (!error) {
+    fetch(DATA_URL).then(response => {
```

Comment:
The normal `Promise` pattern is to do a linear chain instead of nesting, which I think looks cleaner.

```
fetch(DATA_URL)
  .then(response => response.json())
  .then(({features}) => this.setState({
       data: response.features,
       selectedCounty: response.features.find(f => f.properties.name === 'Los Angeles, CA')
       data: features,
       selectedCounty: features.find(f => f.properties.name === 'Los Angeles, CA')
    }));
```